docs/source-app/glossary.rst

```diff
@@ -1,259 +0,0 @@
-:orphan:
```

Comment:
@yurijmikhalevich is this removed on purpose?

Reply:
@Borda, yes. This page is an unused duplicated of "basics.rst" in the same dir.